### PR TITLE
docs: add faq for multi rooms

### PR DESCRIPTION
**Pull request by raviteja83**

### CODE REVIEW

File: docs/javascript/v2/how--to-guides/debugging/faq.mdx [hunk 1]
```diff
@@ -168,3 +168,37 @@ You can get the HLS URL in several ways based on whether you use 100ms client SD
 #### I get type errors related to WebRTC Stats, for example, 'Cannot find name RTCInboundRtpStreamStats'.
 
 The minimum supported version of TypeScript for the SDK is '4.4'. You could update to any version above 4.4 or if you are on an old version of TypeScript and cannot do a major upgrade you can set `skipLibCheck: true` in your tsconfig file.
+
+#### Why is my screenshare not working on mobile browsers?
+
+The screenshare API(getDisplayMedia) is not yet supported on mobile browsers. Refer [this](https://caniuse.com/?search=getDisplayMedia) for supported browsers.
+
+
+#### Can I have multiple instances of HMSRoomProvider each mapping to a different room?
```
Comment (triptu) [inline docs/javascript/v2/how--to-guides/debugging/faq.mdx:177]: i think the question is more of an answer.
suggestion - 

Question - "Can I use the sdk to join multiple rooms at the same time?"

Answer - "Yes. Although one instance of SDK can be used to join only one room, you can create multiple instances of the SDK which can operate independently. 
If you're using react-sdk, you can use multiple instances of HMSRoomProvider. A component using hooks from react-sdk will reach out to the nearest HMSRoomProvider in their ancestor tree. For e.g.
```
const App = () => {

    return (
        <Fragment>
          <HMSRoomProvider>
            <Conference room="room1" />
          </HMSRoomProvider>
          <HMSRoomProvider>
            <Conference room="room2" />
          </HMSRoomProvider>
        </Fragment>
    )
}
```

File: docs/javascript/v2/how--to-guides/debugging/faq.mdx [hunk 2]
```diff
@@ -168,3 +168,44 @@ You can get the HLS URL in several ways based on whether you use 100ms client SD
...
+Room1, Room2 can have the details of the room to which they want to point to
+
+Gotchas:
+
+- The app should have separation between the multiple HMSRoomProvider's like the example above.
+
+
+#### How can I modify noiseSuppression, echoCancellation or autoGainControl on the audio track?
+
+`await hmsActions.setAudioSettings({ advanced:[{ noiseSuppression: false}, { echoCancellation: false }, { autoGainControl: false } ]})`
```
Comment (triptu) [inline docs/javascript/v2/how--to-guides/debugging/faq.mdx:209]: does this actually work? i thought these settings can be configured only at the time of getusermedia

Comment (triptu) [reply docs/javascript/v2/how--to-guides/debugging/faq.mdx:209]: https://bugs.chromium.org/p/chromium/issues/detail?id=796964
https://stackoverflow.com/questions/60873347/applying-constraints-to-an-audio-track-from-getusermedia

Comment (triptu) [reply docs/javascript/v2/how--to-guides/debugging/faq.mdx:209]: or are we internally calling gum again and doing replacetrack?

Comment (raviteja83) [reply docs/javascript/v2/how--to-guides/debugging/faq.mdx:209]: Yes, they do work. we shared this with circle as well. And also peeq. we get a new track when these advanced values are passed.

File: docs/javascript/v2/how--to-guides/debugging/faq.mdx [hunk 3]
```diff
@@ -168,3 +168,41 @@ You can get the HLS URL in several ways based on whether you use 100ms client SD
 #### I get type errors related to WebRTC Stats, for example, 'Cannot find name RTCInboundRtpStreamStats'.
 
 The minimum supported version of TypeScript for the SDK is '4.4'. You could update to any version above 4.4 or if you are on an old version of TypeScript and cannot do a major upgrade you can set `skipLibCheck: true` in your tsconfig file.
+
+#### Why is my screenshare not working on mobile browsers?
+
+The screenshare API(getDisplayMedia) is not yet supported on mobile browsers. Refer [this](https://caniuse.com/?search=getDisplayMedia) for supported browsers.
+
+
+#### Can I use the sdk to join multiple rooms at the same time?
```
Comment (arjunattam) [inline docs/javascript/v2/how--to-guides/debugging/faq.mdx:177]: can this be a standalone page? FAQs don't help in discoverability - too long a list

Comment (raviteja83) [reply docs/javascript/v2/how--to-guides/debugging/faq.mdx:177]: should this be part of join or a separate page itself?

Comment (raviteja83) [reply docs/javascript/v2/how--to-guides/debugging/faq.mdx:177]: as discussed will be added with a demo.

**Review by triptu — APPROVED** [review]

**Pull request merged**